src/main.py

```diff
@@ -32,6 +34,39 @@
 )
 
 
+@app.get(
+    "/",
+    responses={
+        status.HTTP_200_OK: {"description": "Successful response. Returns a context."},
+        status.HTTP_404_NOT_FOUND: {"description": "Not Found. No context found."},
+        status.HTTP_406_NOT_ACCEPTABLE: {
+            "description": "Not Acceptable. The requested format is not supported."
+        },
+    },
+)
+def get_context(
+    request: Request,
+    response: Response,
+    metadata_store: StubMetadataStore = Depends(StubMetadataStore),
+):
+    """
+    Retrieve the context.
+    This endpoint returns information on the context.
+    """
+
+    logger.info("Received request for context")
+    if request.headers["Accept"] == JSONLD or BROWSABLE:
+        context = metadata_store.get_context()
+        if context is not None:
+            response.status_code = status.HTTP_200_OK
+            return context
+        response.status_code = status.HTTP_404_NOT_FOUND
```

Comment:
can you log an error here please (still return a 404, we just want an error logged so we dev types know somethings not right).

for (most of, see next paragraph) the other endpoints a 404 is normal behaviour (they take variables, so someone asking for something that's not there is inevitable and expected). In the case of the context endpoint though, it _should_ always be there as there's no incoming variables, so if its not we'll want to know about it.

to save me raising another ticket could you add an error logging statement for /datasets, /publishers and /topics on the 404 response route as well please while you're in here, same logic.